tools/run_danesfield.py

```diff
@@ -449,8 +450,10 @@ def main(args):
     images_to_use = glob.glob(os.path.join(crop_and_pansharpen_outdir,
                                            "*_crop_pansharpened_processed.tif"))
     orig_meshes = glob.glob(os.path.join(roof_geon_extraction_outdir, "*.obj"))
+
     orig_meshes = [e for e in orig_meshes
-                   if e.find(occlusion_mesh) < 0 and e.find("building_") < 0]
```

Comment:
Seems like we'll need to add a boolean flag to the configuration indicating whether or not "building_"s should be excluded and switch on that

Reply:
we should check the contents of the output in `roof_geon_extraction_outdir`  I think there are some duplicate meshes at that stage, and this was added so that we don't include two copies in the rendering.